GOVERNANCE.md

```diff
@@ -1,37 +1,70 @@
 # Strimzi Governance
 
 This document defines project governance for the Strimzi project.
+
+## Project Roles
+
+### Maintainers
+
+The maintainers are responsible for the overall development of the whole Strimzi project and all its components.
 The maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file. 
 
-## Voting
+#### Changes in Maintainership
 
-The Strimzi project employs voting to ensure no single member can dominate the project. Any maintainer may cast a vote.
+New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority maintainers vote.
+Maintainers can be removed by a ⅔ majority maintainers vote.
 
-For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR.
-Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+#### Github Project Administration
 
-## Changes in Maintainership
+Maintainers will be added to the collaborators list of the Strimzi repository with "Write" access.
+After 6 months a maintainer will be given "Admin" access to the Strimzi repository.
 
-New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority vote.
+### Component owners
 
-Maintainers can be removed by a ⅔ majority vote.
+The component owners are responsible for development of a specific subproject or component within Strimzi.
+Such components might be represented by by a separate GitHub repository within the Strimzi organization (e.g. `strimzi-kafka-bridge`) or by a subdirectory in a GitHub repository (e.g. `./documentation/` in `strimzi-kafka-operator`).
+The components owners are identified in the [`COMPONENT-OWNERS`](COMPONENT-OWNERS) file in this repository which also includes the component for which they are responsible.
+Additionally, each of the components will have `OWNERS` file in its root listing the owners.
```

Comment:
Is it necessary to have both? Seems like a burden.

Reply:
In agreement here, we should just have both sections in the same file.

Reply:
The idea of the `OWNERS` file was to have automated tooling to for example request reviews etc. But since we right now don't have any such tools, I removed it. We can add it later if we have such tools.